src/System.Net.WebHeaderCollection/tests/WebHeaderCollectionTest.cs

```diff
@@ -203,6 +214,18 @@ public void Setter_SameHeaderTwice_Success(string firstName, string secondName)
         }
 
         [Theory]
+        public void Setter()
```

Comment:
Is there a more descriptive name you can pick that's more consistent with the other test names?